### ISSUE TITLE
Vue.js Setup

### ISSUE THREAD

**Issue by JLaveta** [ad440-cloud-project/UX-UI#1]
Create initial Vue.js project and upload to Azure storage blob.

Estimated time: 4 hours

**Comment by JLaveta** [comment]
Created an initial Vue.js project. Uploaded to GitHub repo on a new branch:

https://github.com/ad440-cloud-project/UX-UI/tree/vue-setup

Since TravisCI is not yet integrated, I installed MS Azure Storage Explorer to quickly upload the webpage. A new storage account was created and enabled to handle static web content. Then, I ran "npm run build" on the project folder and uploaded the contents of the "dist" folder.

The next step in this process will be to automate the deployment process using TravisCI.

This initial setup took about 3 hours.
